src/notes/patchV3.6.7.md

```diff
@@ -3,75 +3,75 @@
-- Alarm lights now count for the "Light Vandalism" bingo goal, if they are breakable
-- Fixed mirrored rotation of Limited/Fixed Saves ATM that was added to the gas station, and moved it forwards slightly to get away from some bad collision detection
+- Bingo goals that require the use of drugs or alcohol will no longer be available when those items are banned (e.g. in the Straight Edge loadout).
+- Tweaked weapon placements for different FoVs and ultra-widescreen.
+- Alarm lights now count for the "Light Vandalism" bingo goal, if they are breakable.
+- Fixed mirrored rotation of the Limited/Fixed Saves ATM that was added to the gas station, and moved it forwards slightly to get away from some bad collision detection.
 - Fixed possible crashes with flag names generated for Merchant conversations that exceeded the maximum length.
   - If you have already encountered the Merchant in a map in an in-progress game when you upgrade, he may be a bit confused about what he has available to sell to you.
-- Block certain out-of-bounds locations from being selected for random placement of things (like medbots/repairbots, merchants, turrets, datacubes, etc...)
-  - Outside of the walls of the UNATCO compound in missions 3, 4, and 5
+- Block certain out-of-bounds locations from being selected for random placement of things (like medbots/repairbots, merchants, turrets, datacubes, etc...):
+  - Outside of the walls of the UNATCO compound in Missions 3, 4, and 5
   - Beyond the level transitions between the Wan Chai Market and the Lucky Money
   - Beyond the door to the Vandenberg Computer area from Cmd
-- Removed collision from teleporters so they don't block pawns or items
-- The Bingo Viewer now has an option to always keep the window on top (Under the Display menu)
-- Looking at Weeping Anna will no longer interrupt her death animation
-- For GMDX we now set Hardcore mode when playing on combatDifficulty 4 or higher
-- Fixed randomizing some of GMDX's invisible objects
-- Fixed bug with conversations referencing player's kill counts in M01 and M02
-- Added a timed race to Vandenberg Tunnels (in both the forward and backward directions):  See how fast you can go!
-- UNATCO Troopers that appear in Mission 3, after meeting Lebedev, will no longer hate seeing carcasses (which typically makes them hate the player)
-- The bingo goal for returning to UNATCO HQ in the boat after Mission 2 no longer appears unless the bingo board spans more than 1 mission.
-- Possible fix for some timed races (eg. Catacombs or Reverse Catacombs) where the Mastodon messages were reporting the player losing over 9000% health.
+- Removed collision from teleporters so they don't block pawns or items.
+- The Bingo Viewer now has an option to always keep the window on top (under the Display menu).
+- Looking at Weeping Anna will no longer interrupt her death animation.
+- For GMDX we now set Hardcore mode when starting a game with combat difficulty 4 or higher (Realistic and Impossible).
```

Comment:
The difficulty is called Impossible when playing Rando.  It will show as Realistic if playing Zero Rando mode though, so it's a bit misleading to use an "and" between the choices - they are effectively the same choice.  Maybe the parenthetical could be something like "Such as when playing on Impossible/Realistic difficulty" (Although I imagine most people aren't going to be playing Zero Rando with GMDX)